src/readableview.js

```diff
@@ -0,0 +1,67 @@
+/** @flow */
```

Comment:
It looks like Facebook uses Java-style `CamelCase.js` in all their React apps, so I'm OK doing the same. I didn't see an official Facebook style guide.

GSG definitely doesn't pre-date the idea of classes in JS or files that export a single class. The rationale for `lowercase.js` is that if you're developing on a case-insensitive file system (like a mac), then your site may work locally but not when you deploy (on linux). I don't see that as particularly relevant for us, since we're using an explicit build step.